funcframework/framework.go

```diff
@@ -79,6 +79,11 @@ func registerHTTPFunction(path string, fn func(http.ResponseWriter, *http.Reques
 
 func registerEventFunction(path string, fn interface{}, h *http.ServeMux) {
 	validateEventFunction(fn)
+	defer func() {
+		if r := recover(); r != nil {
+			fmt.Fprintf(os.Stderr, "Validation panic: %v\n\n%s", r, debug.Stack())
```

Comment:
To catch a validation panic, would we need to set up this defer before calling validate?

Reply:
Good point.